lib/tasks/evm_automate.rake

```diff
@@ -10,7 +10,10 @@ module EvmAutomate
   end
 
   def self.simulate(domain, namespace, class_name, instance_name)
+    user = User.where(:userid => 'admin').first
```

Comment:
@kbrock @gmcculloug @tinaafitz 
This might be legacy stuff running simulate from the rake command line. Also in the case of tenants only the admin has access to the appliance console. The simulate is available from the UI and would use the correct user.